server/util/grpc_forward/grpc_forward.go

```diff
@@ -0,0 +1,74 @@
+package grpc_forward
+
+import (
+	"context"
+	"strings"
+	"sync"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/flag"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/mwitkow/grpc-proxy/proxy"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+var (
+	kytheBackendToProxy = flag.String("app.kythe_backend_to_proxy", "", "If set, proxy kythe traffic to this backend")
```

Comment:
nit: the other proxy-ish flags are `-cache_proxy.remote_cache` and `-build_event_proxy.hosts` - maybe `-kythe_proxy.target` would be a good name for this?

Reply:
Ack (removed flag)